data/locale/German.json

```diff
@@ -11,30 +11,30 @@
 
     "Games": "Spiele",
     "All": "Alles",
-    "Press \uE000 to select": "Drücke \uE000 zum auswählen",
-    "\uE001 to return": "\uE001 für zurück",
-    "\uE002 to enter a title ID": "\uE002 zum eingeben der TID",
-    "\uE003 to search": "\uE003 zum suchen",
-    "\uE046 to open the queue": "\uE046 um die Warteschlange zu öffnen",
+    "Press \uE000 to select": "Drücke \uE000 zum Auswählen",
+    "\uE001 to return": "\uE001 , um ins vorherige Menü zu gelangen",
+    "\uE002 to enter a title ID": "\uE002 zum Eingeben einer TID",
+    "\uE003 to search": "Drücke \uE003 zum Suchen",
+    "\uE046 to open the queue": "\uE046 , um die Warteschlange zu öffnen",
 
     "Select a folder:": "Wähle einen Ordner:",
-    "\uE002 to switch to SD": "\uE002 um zu SD zu wechseln",
-    "\uE002 to switch to NAND": "\uE002 um zu NAND zu wechseln",
-    "\uE002 to switch to USB": "\uE002 um zu USB zu wechseln",
+    "\uE002 to switch to SD": "\uE002 , um zur SD-Karte zu wechseln",
+    "\uE002 to switch to NAND": "\uE002 , um zum NAND zu wechseln",
+    "\uE002 to switch to USB": "\uE002 , um zum USB-Stick zu wechseln",
```

Comment:
Don't hardcode this to sticks as HDDs exist, too, and in fact stick usage is discouraged